src/tpm2_wrap.c

```diff
@@ -2560,9 +2582,42 @@ int wolfTPM2_NVCreateAuth(WOLFTPM2_DEV* dev, WOLFTPM2_HANDLE* parent,
         return rc;
     }
 
+    /* Compute NV Index name in case of parameter encryption */
+#ifndef WOLFTPM2_NO_WOLFCRYPT
+    rc = TPM2_GetHashType(in.publicInfo.nvPublic.nameAlg);
```

Comment:
Please move this logic into its own function that takes in an nvPublic struct computes the hash digest (name for NV).

Reply:
Already move into TPM2_HashNvPublic